lib/ssl/src/dtls_connection.erl

```diff
@@ -325,13 +325,12 @@ hello(internal, #hello_verify_request{cookie = Cookie},
     dtls_gen_connection:next_event(?FUNCTION_NAME, no_record, State, Actions);
 hello(internal, #client_hello{extensions = Extensions} = Hello,
       #state{ssl_options = #{handshake := hello},
-             handshake_env = HsEnv,
-             start_or_recv_from = From} = State0) ->
+             handshake_env = #handshake_env{handshake_continue = Continue},
+             start_or_recv_from = From} = State0) when Continue =/= continue ->
```

Comment:
```suggestion
             handshake_env = #handshake_env{handshake_continue = continue},
             start_or_recv_from = From} = State0)  ->
```
If the variable `Continue` only is used in the guard testing for `continue` this can be done directly in the pattern and no guard is needed.